server/routes/serviceProviderReferrals/interventionProgressPresenter.ts

```diff
@@ -112,21 +113,30 @@ export default class InterventionProgressPresenter {
     }
 
     return this.actionPlanAppointments
-      .map(appointment => {
-        const sessionTableParams = this.sessionTableParams(appointment)
-
+      .map((appointment, index, array) => {
+        const isParent =
+          array.indexOf(appointment) === 0 ||
+          (index > 0 && array[index - 1].sessionNumber !== appointment.sessionNumber)
```

Comment:
can we pull this out to a variable, something like isDifferentSessionNumberFromPrevious just to make it a bit clearer what its doing